react-app/src/service.js

```diff
@@ -0,0 +1,15 @@
+import { base } from "./api";
+import axiosInstance from "./axios";
+
+export const getCategories = async () => {
+  try {
+    const { data } = await axiosInstance({
+      method: "get",
+      url: base + "/categories",
+    });
+
+    return data;
+  } catch (error) {
+    console.log("not successful");
+  }
+};
```

Comment:
This is a very good practice to create services and do all your API calls in one place, then export and import them where needed. To make sure you have a single source of truth and you can easily change the endpoint in one place and it will reflect everywhere you've used it if say backend changed the name of an endpoint, which is very possible.